src/Api/Controllers/AccountsController.cs

```diff
@@ -194,6 +195,30 @@ public async Task PostPassword([FromBody]PasswordRequestModel model)
             await Task.Delay(2000);
             throw new BadRequestException(ModelState);
         }
+        
+        [HttpPost("password-no-compare")]
```

Comment:
Let's just call this `set-password`

Reply:
Should that be a `PUT password` instead? Not sure the "pure" REST convention for something like this, but otherwise, yeah, POST set-password seems good.

Reply:
Updated name in the controller/request model.